### PR TITLE
Add missing west module entries (Renesas and Wurth)

### PR DESCRIPTION
**Pull request by fabiobaltieri**
Add two missing module maintainer entries, links for the respective HAL issues in the two commits.

### CODE REVIEW

**Comment by fabiobaltieri** [comment]
cc @mah-eiSmart (can't add to reviewers)

File: MAINTAINERS.yml [hunk 1]
```diff
@@ -2357,6 +2365,15 @@ West:
...
     - manifest-hal_ti
 
+"West project: hal_wurthelektronik":
+  status: odd fixes
+  collaborators:
+    - mah-eiSmart
+  files:
+    - modules/Kconfig.wurthelektronik
+  labels:
+    - manifest-hal_wurthelektronik
```
Comment (stephanosio) [inline MAINTAINERS.yml:2375]: why is this "odd fixes"? based on https://github.com/zephyrproject-rtos/zephyr/issues/47021, @mah-eiSmart should be maintaining it?

Comment (fabiobaltieri) [reply MAINTAINERS.yml:2375]: Sorry, copy paste, let me change it. Thanks for catching it.

File: MAINTAINERS.yml [hunk 2]
```diff
@@ -2357,6 +2365,15 @@ West:
   labels:
     - manifest-hal_ti
 
+"West project: hal_wurthelektronik":
+  status: maintained
+  collaborators:
+    - mah-eiSmart
```
Comment (stephanosio) [inline MAINTAINERS.yml:2371]: If it is maintained, there needs to be a maintainer.

Comment (fabiobaltieri) [reply MAINTAINERS.yml]: I need more coffee.

**Review by stephanosio — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**